core/celo_genesis.go

```diff
@@ -49,7 +56,27 @@ func celoGenesisAccounts() map[common.Address]GenesisAccount {
 	if err != nil {
 		panic(err)
 	}
+	sortedOraclesBytecodeLinked := bytes.Replace(contracts.SortedOraclesBytecodeRaw, []byte("__$c0b499b413513d0c67e2a6a17d90846cb3$__"), []byte("000000000000000000000000000000000000ce17"), -1)
```

Comment:
Will we have to adapt this every time the bytecode changes?

Reply:
I failed at looking up that information. I assume it is something stable like `keccak("AddressSortedLinkedListWithMedian")` to get a fixed length placeholder.